aclk/agent_cloud_link.c

```diff
@@ -819,24 +819,39 @@ void aclk_get_challenge(char *aclk_hostname, char *aclk_port)
 #pragma endregion
 #endif
 
-static void aclk_try_to_connect(char *hostname, char *port, int port_num)
+static void aclk_try_to_connect(char *hostname, int port)
 {
+    int rc;
+
+// this is usefull for developers working on ACLK
+// allows connecting agent to any MQTT broker
+// for debugging, development and testing purposes
+#ifndef ACLK_DISABLE_CHALLENGE
```

Comment:
You should add an `#else` clause that prints a huge `error()` message with all capitals warning the user that it's an insecure build and it's not supported by netdata.

Reply:
Will add. No worries agent built like this is not able to connect to Cloud though.